config.lua

```diff
@@ -15,15 +15,15 @@ Config.AuthorizedVehicles = {
 	car = {
 		crew = {},
 
-		cameraman = {
+		cam_man = {
 			{model = 'rumpo', price = 20000}
 		},
 
-		reporter = {
+		report = {
```

Comment:
See
https://github.com/lockdownstudio/esx_journalist/blob/17947aad0717575854d6250436465a117d64f476/client/main.lua#L140

This changes will throw error on the client script.